nss-cohort-36/bangazon-client-virtual-magicians-1
# Jg product 3

# Description
YAY! You can now post products to our data base!
## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-product-3`
4.Please run `python manage.py runserver` and `npm start` before running your tests in the appropriate areas in your terminal.
5. go to our site and select the `sell` section on the nav. Fill out the form and you should see the magic (in your db table on TablePlus.
# Checklist:
